### PR TITLE
Enhancement: User Panel

### PR DESCRIPTION
**Pull request by ins0**
this pr will

- [x] trigger the modules ajax request only when the user navigates to the correspondending tab
- [x] view fixes
- [x] added more string escaping to unsafe vars

### CODE REVIEW

File: module/User/view/zfc-user/user/index.phtml [hunk 1]
```diff
@@ -1,3 +1,17 @@
+<?php
+$this->inlineScript()->appendScript(<<<EOT
```
Comment (Ocramius) [inline module/User/view/zfc-user/user/index.phtml]: note that you should use more explicit names, such as `<<<JAVASCRIPT`. Also use nowdoc, not heredoc:

`$this->inlineScript()->appendScript(sprintf($template, $this->escapeJs($this->url('zf-module'))));`

**Comment by Ocramius** [comment]
> trigger the modules ajax request only when the user navigates to the correspondending tab

Is this actually an improvement? I'd rather either have a separate page or still have the content.

**Comment by ins0** [comment]
i would say so as the outcome is the same as before just saves api calls if a user is logged in but don't hit the repository tab. but after the outcome of #349 this will be replaced in a future version and is minor

**Comment by ins0** [comment]
will close as this is outdated and may improved on different way

**Comment by localheinz** [comment]
@ins0 

What do you think, shall we update the actual design of the site as well? I think it could be improved a lot!

**Comment by ins0** [comment]
@localheinz i'm not skilled in creating redesigns/ux design - but why not and give it a try :8ball: yep

**Comment by localheinz** [comment]
@ins0 

:+1:

**Comment by Ocramius** [comment]
:8ball: good luck!

**Pull request closed without merging**